src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/XmlDataContract.cs

```diff
@@ -412,6 +435,29 @@ private bool RequiresMemberAccessForCreate(SecurityException securityException)
         }
 #endif
 
+        internal IXmlSerializable ReflectionCreateXmlSerializable(Type type)
+        {
+            if (type.GetTypeInfo().IsValueType)
+            {
+                throw new NotImplementedException("ReflectionCreateXmlSerializable - value type");
+            }
+            else
+            {
+                object o = null;
+                if (type == typeof(System.Xml.Linq.XElement))
+                {
+                    o = new System.Xml.Linq.XElement("default");
+                }
+                else
+                {
+                    ConstructorInfo ctor = GetConstructor();
```

Comment:
Is XElement the only type where the default ctor is not available or correct?  What is the expected behavior if a type does not have a default ctor or is abstract?